samples/keys/identity_key_generation/prj.conf

```diff
@@ -0,0 +1,31 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+CONFIG_MAIN_STACK_SIZE=8192
+
+CONFIG_CONSOLE=y
+CONFIG_LOG=y
+
+# The HW_UNIQUE_KEY library is needed for the MKEK
+CONFIG_HW_UNIQUE_KEY=y
+CONFIG_HW_UNIQUE_KEY_RANDOM=y
+
+CONFIG_IDENTITY_KEY=y
+CONFIG_IDENTITY_KEY_RANDOM=y
+
+# NRF_SECURITY is needed to generate a random identity key
+CONFIG_NRF_SECURITY=y
+CONFIG_PSA_CRYPTO_DRIVER_CC3XX=y
+CONFIG_PSA_CRYPTO_DRIVER_OBERON=n
+CONFIG_PSA_CRYPTO_DRIVER_BUILTIN=n
+CONFIG_MBEDTLS_PSA_CRYPTO_C=y
+CONFIG_MBEDTLS_ENABLE_HEAP=y
+CONFIG_MBEDTLS_HEAP_SIZE=8192
+
+CONFIG_PM_SINGLE_IMAGE=y
```

Comment:
Is this necessary? Doesn't PM know that this is a single image?

Reply:
Removed usage of PM

Fixed